docs/connector-development/config-based/understanding-the-yaml-file/declarative_oauth_examples/current_example.yml

```diff
@@ -0,0 +1,173 @@
+version: 6.13.0
+
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  stream_names:
+    - moves
+
+definitions:
+  streams:
+    moves:
+      type: DeclarativeStream
+      name: moves
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /api/v2/move/
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestPath
+          pagination_strategy:
+            type: CursorPagination
+            cursor_value: "{{ response.get('next') }}"
+            stop_condition: "{{ response.get('next') is none }}"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/moves"
+  base_requester:
+    type: HttpRequester
+    url_base: https://pokeapi.co
+    authenticator:
+      type: OAuthAuthenticator
+      refresh_request_body: {}
+      grant_type: refresh_token
+      client_id: '{{ config["client_id"] }}'
+      client_secret: '{{ config["client_secret"] }}'
+      refresh_token: '{{ config["client_refresh_token"] }}'
+      access_token_value: '{{ config["client_access_token"] }}'
+
+streams:
+  - $ref: "#/definitions/streams/moves"
+
+spec:
+  type: Spec
+  connection_specification:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    required:
+      - client_id
+      - client_secret
+      - client_access_token
+    properties:
+      client_id:
+        type: string
+        title: Client ID
+        airbyte_secret: true
+        order: 0
+      client_secret:
+        type: string
+        title: Client secret
+        airbyte_secret: true
+        order: 1
+      client_access_token:
+        type: string
+        title: Access token
+        airbyte_secret: true
+        airbyte_hidden: false
+        order: 2
+    additionalProperties: true
+  advanced_auth:
+    auth_flow_type: oauth2.0
+    oauth_config_specification:
+      oauth_connector_input_specification:
+        consent_url: >-
+          https://yourconnectorservice.com/oauth/consent?client_id={{client_id_value}}&redirect_uri={{
+          redirect_uri }}&state={{ state }}
```

Comment:
nit: redirect_uri={{ redirect_uri_value }}

Note, the `redirect_uri_value` is not `urlEndoded` by default. To make it `urlEncoded` we do `{{ {{ redirect_uri_value}} | urlEncoder }}`